nginx.conf

```diff
@@ -64,7 +64,7 @@ http {
 	proxy_set_header X-Forwarded-For $remote_addr;
 	proxy_set_header X-Forwarded-Proto $scheme;
 	proxy_set_header X-Request-Id $request_id;
-	proxy_read_timeout 30s;
+	proxy_read_timeout 35s;
```

Comment:
Should this value be in sycn with the `harakiri` value below? If so we should at least document this if not find a way to link them through an env variable if possible.

Reply:
Values are based on [zulip/zulip#31909](https://github.com/zulip/zulip/pull/31909).

Reply:
Ah, so they _are_ related and we should document the relationship if we cannot automate it somehow.